frontend/src/components/Reward.tsx

```
@@ -0,0 +1,34 @@
+    gems: number;
+    onContinue: () => void;
+}
+
+export default function Reward({ title, gems, onContinue }: RewardProps) {
+    const handleContinue = async () => {
+        onContinue();
+    };
+
+    return (
+        <>
+            <div className={styles.overlay}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)